pom.xml

```diff
@@ -297,6 +297,29 @@
               <version>${gax.version}</version>
               <classifier>testlib</classifier>
             </dependency>
+            <dependency>
+              <groupId>com.google.cloud</groupId>
+              <artifactId>google-cloud-conformance-tests</artifactId>
+              <version>0.0.1</version>
```

Comment:
This is marked as `test` in the modules pom where it's declared as a `dependency` rather than here in the `dependencyManagement`. I can move it here if you'd rather it be here?

Reply:
I would prefer to keep the dependencyManagement solely focused on version management. I think the leaf modules should decide on the scope

Reply:
Sounds good, I'll keep it down in the module.